src/proto/proto/db3_node.proto

```diff
@@ -64,18 +64,43 @@ message GetKeyResponse {
     bytes signature = 1;
     BatchGetValue batch_get_values = 2;
 }
+message QuerySessionInfo {
+    int32 session = 1;
+    int32 session_query_count = 2;
+}
+message RestartSessionRequest {
+    bytes query_session_info = 1;
+    bytes signature = 2;
+}
+message RestartSessionResponse {
+    int32 session = 3;
+}
 
 message GetAccountRequest {
     // the hex encoded string
     string addr = 1;
 }
 
+message GetSessionInfoRequest {
+    // the hex encoded string
+    string addr = 1;
+}
+message GetSessionInfoResponse{
+    // the hex encoded string
+    int32 id = 1;
+    int64 start_time = 2;
+    string status = 3;
+    int32 query_count = 4;
 
+}
 service StorageNode {
     // method for querying bills by height
     rpc QueryBill(QueryBillRequest) returns (QueryBillResponse) {}
     // method for querying kv data
     rpc GetKey(GetKeyRequest) returns (GetKeyResponse) {}
+    // method for restart query session
+    rpc RestartQuerySession(RestartSessionRequest) returns (RestartSessionResponse) {}
```

Comment:
I prefer to use `open session`  and `close session` to replace the `restart session`

Reply:
Maybe one client can have two or three sessions at the same time

Reply:
will fix in the next pr